docs/source/new-docs/getting-started/best-practices/testing.rst

```diff
@@ -0,0 +1,11 @@
+:orphan:
+
+Testing
+=======
+
+.. note::
+
+    Setting up WSL through windows is the easiest method for running GeoIPS on
+    a Windows system. To do so, follow directions on
+    `Microsoft WSL Install <https://learn.microsoft.com/en-us/windows/wsl/install>`_.
+    This requires administrator privileges.
```

Comment:
What is the intention for this section of the getting started?

Reply:
The intention is to add more information about testing geoips in the future, but that's not needed here